docs/content/latest/api/ysql/datatypes/type_array/functions-operators/properties.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Array properties
+linkTitle: Array properties
+headerTitle: Functions for reporting the geometric properties of an array
+description: Functions for reporting the geometric properties of an array
+menu:
+  latest:
+    identifier: array-properties
+    parent: array-functions-operators
+isTocNested: false
+showAsideToc: false
+---
+
+These functions return the various dimensional properties that jointly characterize an array. These three together completely specify the shape and size:
+
+- its dimensionality (`int`) ;
+- the lower bound along each dimension (`int`);
+- the upper bound along each dimension (`int`).
```

Comment:
```suggestion
- its dimensionality (`int`).
- the lower bound along each dimension (`int`).
- the upper bound along each dimension (`int`).
```

Reply:
When you have a list of bullets, each of which is a phrase (and not a sentence), what rule tells you to start each point with lower case letter and finish it with a period? I see this use of bullets as nothing other than a visual device that makes the old-fashioned way easier to read—namely a colon followed by phrases ending each with a semicolon (until the last ends with a period). I'm accepting this for now. But I hate it.